osidb/tests/test_flaw.py

```diff
@@ -1890,7 +1906,11 @@ def test_validate_historical_affectedness_resolution(
 
         affect.affectedness = affectedness_new
         affect.resolution = resolution_new
-        affect.impact = Impact.CRITICAL
+        affect.impact = (
+            Impact.LOW
+            if resolution_new == Affect.AffectResolution.DEFER
+            else Impact.CRITICAL
+        )
```

Comment:
There does not seem to be a reason for any specific impact so LOW all the time should be OK

Reply:
Removed in light of not needing validation.